lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -546,7 +546,12 @@ module.exports = class MethodMappings {
 
           return elementText;
         },
+        async isVisibleElement(id) {
+          const element = await this.getWebElement(id);
+          const elementVisible = await element.isDisplayed();
 
+          return elementVisible;
+        },
```

Comment:
We don't need to create this again, there's already a `isElementDisplayed` method which does the same.

Reply:
Oh i didnt notice **isElementDisplayed** because i was taking reference from **getText()**